src/main/resources/application.yml

```diff
@@ -198,6 +198,10 @@ ide-sidecar:
     max-backoff-ms: 5000
     timeout-ms: 10000
     max-retries: 5
+  integration-tests:
+    # cp-demo configs used by Confluent Platform integration tests
+    cp-demo:
+      tag: v7.7.1
```

Comment:
Does the this version or the one in the makefile take precedence?

Reply:
The Makefile parses the version defined here using `yq`.